### DISCUSSION TITLE
How critical is it to have a buzzer housing?

### DISCUSSION THREAD

**Discussion by TactileDesign** [kriswilk/buzzah#18] [Q&A]
In the design I have been using, I have ERMs that are held in place by wire ties.  The tightness of my glove keeps the ERM against my fingertips.  This seems to work OK for me and I have definately been getting therapudic relief since starting.   My gloves are slender enough that I can reach into my pocket and remove my keys.  But, I have noticed that several people here and elsewhere make housings for their buzzers that have somewhat smaller contact area than the entire buzzer face - more similar to the actual Tass buzzer design.  My question is, How critical is it to incorporate the plastic housing?  Is the additional efficacy of the buzzer within a housing worth the trade off with dexterity given that we need to spend so many hours per day in therapy?

**Comment by Neurosmith24** [comment]
In Tass' published studies he focused the stimulation of each fingertip to a 5mm circle. So, housings come into play to hold the actuator within a housing that the finger can rest on with the (theoretical) only contact with the finger being a 5mm disc that sits on top of the LRA/ERM. 

Tass' studies also specify that the vibration be perpendicular to the skin which is why many people focus on LRAs vs ERMs. So, how critical is it? Few besides Dr. Tass would be able to answer that with authority. But if one is trying their best to mimic Tass' work then it would be important. That said, there are plenty of folks out there who use ERMs that don't have a focusing disc and are realizing benefits.

**Reply by TactileDesign** [reply]
Thanks for your response!  Also, many many MANY thanks for making your Buzzah design available to all.  It is really fantastic that it involves little to no electronics skill to assemble - making this technology available to PWP who dont happen to have a background in electronics.

**Reply by Neurosmith24** [reply]
@kriswilk gets all the thanks for creating the Buzzah design. I

**Reply by TactileDesign** [reply]
Sorry!  I somehow thought you were he....

**Reply by kriswilk** [reply]
Thanks for the mention, @Neurosmith24. However, at this point those of you who have built and are using these devices have more practical experience than I do. I also don't have the time to keep up with the latest research, so I appreciate all the input from others here!
